client/dataset_definitions_test.go

```diff
@@ -4,28 +4,60 @@ import (
 	"context"
 	"testing"
 
+	"github.com/honeycombio/terraform-provider-honeycombio/internal/helper/test"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
 func TestDatasetDefinitions(t *testing.T) {
+	t.Parallel()
+
 	ctx := context.Background()
 
 	c := newTestClient(t)
 	dataset := testDataset(t)
+	definitionDefaults := DatasetDefinitionDefaults()
+
+	// ensure default definition columns exist -- create any which may be missing.
```

Comment:
Does this mean we can remove some or all of required columns section from the setup script? https://github.com/honeycombio/terraform-provider-honeycombio/blob/main/scripts/setup-testsuite-dataset#L11

Reply:
We're not quite there yet, I don't think. I need to do a deeper pass in the provider code to make sure we're not dependant on any of them.